extensions/wikia/IndexingPipeline/PipelineEventProducer.class.php

```diff
@@ -10,17 +11,18 @@ class PipelineEventProducer {
 	const ACTION_CREATE = 'create';
 	const ACTION_UPDATE = 'update';
 	const ACTION_DELETE = 'delete';
-	/** @var PipelineConnectionBase */
-	protected static $pipe;
+
+	public function __construct() {}
 
 	/**
 	 * @desc Send event to pipeline in old format (message with params inside).
 	 *
-	 * @param $eventName
 	 * @param $pageId
+	 * @param null $revisionId
+	 * @param null $eventName
 	 * @param array $params
 	 */
-	public static function send( $eventName, $pageId, $revisionId, $params = [ ] ) {
+	public static function send( $pageId, $revisionId, $eventName, $params = [ ] ) {
```

Comment:
This is a backwards - incompatible change ( see:
https://github.com/Wikia/app/pull/9325/files#diff-51080aaef54b01dd7a86c59c827a2761R110
and related)